### ISSUE TITLE
Jasmine typings

### ISSUE THREAD

**Issue by klinki** [DefinitelyTyped/DefinitelyTyped#8927]
- [x] I tried using the latest `jasmine/jasmine.d.ts` file in this repo and had problems.

Hi guys,
I'm having problems with jasmine typings. I'm using Jasmine 2.4 and I want to use createSpy functionality. I noticed it is nested in jasmine module in typings, so I have to write 
`jasmine.createSpy` or import `createSpy = jasmine.createSpy;`

I had some issues when using it with karma - it could not load the createSpy function was throwing Reference error createSpy is not defined. (https://github.com/karma-runner/karma-jasmine/issues/114)

When I used createSpy without import and without jasmine prefix, everything worked fine. But TypeScript compiler was complaining 

Is there any reason, why some functions are nested in jasmine module and some are not?

**Comment by vvakame** [comment]
cc/ @borisyankov @theodorejb @davidparsson

IMO, you should try with javascript first. not typescript.
http://jasmine.github.io/2.4/introduction.html#section-Spies:_<code>createSpy</code>

**Comment by klinki** [comment]
Good point, I didn't notice that :(  OK, Typings are correct then. I got confused, because I had those issues I explained on karma-runner and I couldn't figure it out. Everything started working when I stopped importing createSpy. Only issues I had were with TypeScript compiler, so I thought it might be error in definitions.

Sorry for blaming the definitions, I will have to find some other solution...
